presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -142,6 +156,37 @@ public void testSchemaOperations()
         assertUpdate("DROP SCHEMA new_schema");
     }
 
+    @Test
+    public void testIOExplain()
+    {
+        assertUpdate("CREATE TABLE test_orders (LIKE orders INCLUDING PROPERTIES)");
+
+        String query = "INSERT INTO test_orders SELECT * FROM orders where orderkey > 5";
+        MaterializedResult result = computeActual("EXPLAIN (TYPE IO) " + query);
+        TableColumnInfo input = new TableColumnInfo(
+                new QualifiedTablePrefix("hive", "tpch", "orders"),
+                ImmutableList.of(
+                        new ColumnConstraint(
+                                "orderkey",
+                                BIGINT,
+                                new FormattedDomain(
+                                        false,
+                                        ImmutableList.of(
+                                                new FormattedRange(
+                                                        new FormattedMarker(Optional.of("5"), ABOVE),
+                                                        new FormattedMarker(Optional.empty(), BELOW))))
+                        )
+                )
+        );
+        assertEquals(
+                getOnlyElement(result.getOnlyColumnAsSet()),
+                jsonCodec(IOPlan.class).toJson(new IOPlan(
```

Comment:
I understand there is current need for machine readable output.
However, this functionality is available to end users reaching presto with CLI. They can execute & be able to read the results of:
- EXPLAIN <query>
- EXPLAIN (TYPE DISTRIBUTED) <query>
- EXPLAIN ANALYZE <query>
 
and now
- EXPLAIN (TYPE IO) <query>

expected that the last doesn't print human readable output, while can still be very useful for users.

I think we should have:
- EXPLAIN (TYPE IO) <query> -- printing the information in a human readable way
- provide IO information elsewhere, as we do with other execution stats

Reply:
What about this: `EXPLAIN (TYPE IO, FORMAT JSON)`?

We already have `FORMAT TEXT` (default) and `FORMAT GRAPHVIZ`. For `TYPE IO`, we'll just fail unless it's `FORMAT JSON` for now.

cc @martint